applications/NXscan.nxdl.xml

```diff
@@ -27,6 +27,21 @@
 	    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
 	    >
+    <symbols>
+        <doc>
+		    The symbol(s) listed here will be used below to coordinate datasets
+			with the same shape.
```

Comment:
make the indentation consistent here

Reply:
I think what I will do is run each application definition through an XML formatter so that they will all be consistent, hopefully Travis wont mind